examples/large_models/llama-fairscale/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Loading large Huggingface models with constrained resources using accelerate
+
+
+This document briefs on serving the [Llama 2](https://huggingface.co/meta-llama) as presented in the original [Llama repo](https://github.com/facebookresearch/llama/tree/main) using FairScale Tensor Parallel. It basically, is relying on Tensor Parallel layers that build the model, more of Megatron Style. In the following, we show the steps how to serve the 7-70B  model with Torchserve.
+
+### Step 1: Download model
```

Comment:
Will be better to give steps to download from Meta llama site: https://ai.meta.com/llama/ or point to llama GH for steps to download: https://github.com/facebookresearch/llama#download